Clean up code to work with gcc4

patch by Mark Loeser

--- allegro-4.1.18/src/gui.c
+++ allegro-4.1.18/src/gui.c
@@ -1798,6 +1798,7 @@
 
 
 
+static int shutdown_single_menu(MENU_PLAYER *, int *);
 /* update_menu:
  *  Updates the status of a menu player object returned by init_menu(),
  *  returning TRUE if it is still active or FALSE if it has finished.
@@ -1810,8 +1810,6 @@
  */
 int update_menu(MENU_PLAYER *player)
 {
-   static int shutdown_single_menu(MENU_PLAYER *, int *);
-
    MENU_PLAYER *i;
    int c, c2;
    int old_sel, child_ret;
--- allegro-4.1.18/src/sound.c
+++ allegro-4.1.18/src/sound.c
@@ -25,7 +25,7 @@
 
 
 
-extern DIGI_DRIVER digi_none;
+static DIGI_DRIVER digi_none;
 
 
 

